tests/install.test.js

```diff
@@ -1,15 +1,17 @@
 require('chai').should();
+var execSync = require('child_process').execSync;
 var gitHooks = require('../lib/git-hooks');
 var fsHelpers = require('../lib/fs-helpers');
 
-var SANDBOX_PATH = __dirname + '/tmp-sandbox/';
+var SANDBOX_PATH = '/tmp/tmp-sandbox/';
```

Comment:
Let's use `mktemp -d` here. It creates a temporary directory and returs its name.